auto_archive.py

```diff
@@ -86,7 +92,7 @@ def process_sheet(sheet, header=1, columns=GWorksheet.COLUMN_NAMES):
 
     # loop through worksheets to check
     for ii, wks in enumerate(sh.worksheets()):
-        logger.info(f'Opening worksheet {ii}: "{wks.title}" header={header}')
+        logger.info(f'Opening worksheet ii={ii}: {wks.title} header={header}')
```

Comment:
```suggestion
        logger.info(f'Opening worksheet {ii=}: {wks.title=} {header=}')
```
this will get you the same result, see [py3.8 release notes](https://docs.python.org/3/whatsnew/3.8.html#f-strings-support-for-self-documenting-expressions-and-debugging)